README.md

```diff
@@ -1,39 +1,58 @@
-Kubernetes deployment of RADAR-base.
+# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=dev)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
+The repository contains Helm charts and other configuration needed to install RADAR-base on a Kubernetes cluster.
 
+## Prerequisites
+#### Kubernetes cluster 
+You need to have a working Kubernetes installation and there are 3 ways to have that:
+* From cloud providers such as: [AWS](https://aws.amazon.com/eks/), [GCP](https://cloud.google.com/kubernetes-engine/), [Azure](https://docs.microsoft.com/en-us/azure/aks/), etc
+* Install it on your own servers with: [Rancher](https://rancher.com/), [Kubespray](https://github.com/kubernetes-sigs/kubespray), [Kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/), [OpenShift](https://www.okd.io/), etc
+* Install it on your local machine with: [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/), [K3S](https://k3s.io/), etc
+
+If you're not using a cloud provider you need to make sure that you can [load balance](https://kubernetes.github.io/ingress-nginx/deploy/baremetal/), [expose  applications](https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/#exposing-the-service) and provide [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+
+#### DNS server
+Some applications are only accessible via HTTPS and it's essential to have a DNS server via providers like GoDaddy, Route53, etc
+
+#### SMTP Server
+RADAR-Base needs an SMTP server to send registration email to researchers and participants.
+
+#### Dockerhub account
+Due to introduction of [Dockerhub download rate limiter ](https://docs.docker.com/docker-hub/download-rate-limit/) and number of images used in RADAR-Kubernetes installation you might hit the limit size pretty quickly so it's advised to at least use an authenticated account or purchase a subscription. You can also use a third party container registry to cache RADAR-Base images and avoid using Dockerhub.
+
+#### AWS KMS
+
+
+#### (Optional) Object storage
+An external object storage allows RADAR-Kubernetes to backup cluster data such as manifests, application configuration and data via [Velero](https://velero.io/) to a backup site.
+
+You can also send the RADAR-Base output data to this object storage, which can provider easier management and access compared to bundled [Minio](https://min.io/) server inside RADAR-Kubernetes.
+
+#### (Optional) Managed services
+Radar-Kubernetes provides Kafka cluster, PostgreSQL server, Object storage, Log server and Monitoring system inside the cluster but you can disable these bundled  components and use an externally managed services like Confluent Cloud, Azure Database for PostgreSQL, AWS S3, etc instead.
+
+#### (Optional) Firebase account
+Firebase account is needed in case you want to use RADAR-Base mobile apps to collect data.
```

Comment:
Only needed if you customise the apps yourself. I think it can be removed from here and check that we have it documented at the app side.